app/javascript/packages/document-capture/components/unknown-error.tsx

```diff
@@ -0,0 +1,47 @@
+import type { ComponentProps } from 'react';
+import { ReactNode } from 'react';
+import { formatHTML, useI18n } from '@18f/identity-react-i18n';
+import { FormStepError } from '@18f/identity-form-steps';
+
+interface UnknownErrorProps extends ComponentProps<'p'> {
+  unknownFieldErrors: FormStepError<{ front: string; back: string }>[];
+  isFailedDocType: boolean;
+  remainingAttempts: number;
+  altFailedDocTypeMsg?: string | null;
+}
+
+function formatWithStrongNoWrap(text: string): ReactNode {
+  return formatHTML(text, {
+    strong: ({ children }) => <strong className="text-no-wrap">{children}</strong>,
+  });
+}
```

Comment:
this is basically a Component right?
```suggestion
function HtmlTextWithStrongNoWrap({ text: string }): ReactNode {
  return <>
    {formatHTML(text, {
      strong: ({ children }) => <strong className="text-no-wrap">{children}</strong>,
    })}
  </>;
}
```
(plus or minus some ESLint)

Then usage is:
```jsx
<HtmlTextWithStrongNoWrap text={text} />
```

Reply:
@zachmargolis , moved it out, since there are other places using similar functionalities(review-issues-step.tsx).